psamm/datasource/native.py

```diff
@@ -413,18 +423,6 @@ def parse_compound_yaml_file(path, f):
     return parse_compound_list(path, yaml_load(f))
 
 
-def resolve_format(format, path):
-    """Looks at a file's extension and format (if any) and returns format.
-    """
-    if format is None:
-        if (re.match(r'.+\.(yml|yaml)$', path)):
-            return 'yaml'
-        elif (re.match(r'.+\.tsv$', path)):
-            return 'tsv'
-    else:
-        return format.lower()
-
-
```

Comment:
This change and the following ones revert the `resolve_format` changes from another pull request. This must be a mistake.